pkgs/build-support/trivial-builders/test/writeStringReferencesToFile.nix

```diff
@@ -1,9 +1,11 @@
-{ callPackage, lib, pkgs, runCommand, writeText, writeStringReferencesToFile }:
+{ callPackage, lib, pkgs, runCommand, writeText, concatText, writeStringReferencesToFile }:
```

Comment:
This is the writeStringReferencesToFile test.

Please create a separate file in this directory and a new attribute in `pkgs/test/default.nix`.